charts/consul/templates/cni-daemonset.yaml

```diff
@@ -0,0 +1,91 @@
+{{- if .Values.global.imageK8s }}{{ fail "global.imageK8s is not a valid key, use global.imageK8S (note the capital 'S')" }}{{ end -}}
```

Comment:
why do we need the image check here?

Reply:
I was just copying the pattern from another template. I guess that one will catch it and this is redundant. I shall remove it in the couple places that I had it.